platform/device/device.go

```diff
@@ -38,6 +38,7 @@ type Device struct {
 	DEPProfileAssignedDate time.Time        `db:"dep_profile_assigned_date"`
 	DEPProfileAssignedBy   string           `db:"dep_profile_assigned_by"`
 	LastSeen               time.Time        `db:"last_seen"`
+	BootstrapToken         string           `db:"bootstrap_token"`
```

Comment:
Any reason this, too, shouldn't be a byte slice? I see a few `string()` and `[]byte()` type conversions and I wonder if they're necessary.

Reply:
I mainly left these as is because it matches existing code (namely, Token and Unlock Token are also stored as strings). I agree it would make more sense to store it as a byte (same in the protobuf), but figured consistency with the rest of the codebase was better. []byte-string conversions cost very little I would think. If you want to move it to a []byte, I can make those changes (proto file, generated code, services, etc). Just let me know.

Reply:
Sure, sounds good. :)